### PR TITLE
WRN-12748: Update eslint 8 related modules

### PR DESCRIPTION
**Pull request by hong6316**
### Issue Resolved / Feature Added
ESLint v8.0.0 is [released](https://eslint.org/blog/2021/10/eslint-v8.0.0-released) 🎉
Dependencies should be compatible with ESLint 8 too

### Resolution
update eslint 8 & related modules

### Additional Considerations
**※ BEFORE THIS CHANGE, UPDATE & RELEASE WRN-12124 FIRST.**

### Links
WRN-12748

### Comments
Enact-DCO-1.0-Signed-off-by: Taeyoung Hong (taeyoung.hong@lge.com)

### CODE REVIEW

**Review by MikyungKim — COMMENTED** [review]
Please update your branch with the latest base branch and
Please update other `eslint-plugin-*` modules.

**Comment by hong6316** [comment]
@MikyungKim ,
> Please update your branch with the latest base branch and Please update other `eslint-plugin-*` modules.

OK, every `eslint-plugin-*` are the latest version

File: package.json [hunk 1]
```diff
@@ -59,7 +59,7 @@
     }
   },
   "devDependencies": {
-    "eslint": "^7.30.0",
+    "eslint": "^8.11.0",
```
Comment (MikyungKim) [inline package.json:62]:
```suggestion
    "eslint": "^8.12.0",
```

**Review by MikyungKim — APPROVED** [review]
:lgtm:

**Pull request merged**